suite-native/module-device-settings/src/hooks/useChangeDeviceName/DeviceNameFormSchema.tsx

```diff
@@ -0,0 +1,25 @@
+import { yup } from '@suite-common/validators';
+import { useTranslate } from '@suite-native/intl';
+import { isAscii } from '@trezor/utils';
+
+const noSpecialPattern = /^(?!.*[\p{M}\p{Lm}])[\x20-\x7E\p{L}\p{N}'-]+$/u;
```

Comment:
```suggestion
const noSpecialCharacter = /^(?!.*[\p{M}\p{Lm}])[\x20-\x7E\p{L}\p{N}'-]+$/u;
```

Maybe this would be a better name? It's probably opinionated, feel free to resolve.

Reply:
https://github.com/trezor/trezor-suite/commit/f912231189f29ade5224d7da3831a45085953ca2